core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/compare/ConceptMapCompareDsvExporter.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 B2i Healthcare Pte Ltd, http://b2i.sg
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.b2international.snowowl.core.compare;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import com.b2international.snowowl.core.date.Dates;
+import com.b2international.snowowl.core.domain.ConceptMapMapping;
+import com.fasterxml.jackson.dataformat.csv.CsvMapper;
+import com.fasterxml.jackson.dataformat.csv.CsvSchema;
+
+/**
+ * @since 7.13
+ */
+public final class ConceptMapCompareDsvExporter {
+	
+	private static final char DELIMITER = ';';
```

Comment:
Since you have introduced this as a new API, I suggest converting it to a Request implementation and make the following arguments configurable:
* filePath
* delimiter
* changeKinds